tasks/prime.py

```diff
@@ -378,6 +379,29 @@ def on_start(self) -> None:
         # Set local variables with needed info to create a move
         self.set_default_mto_ids(move_id)
 
+    @tag(MOVE_TASK_ORDER, "getMoves")
+    @task(2)
+    def get_moves(self) -> None:
```

Comment:
Thanks for adding this back, was definitely an oversight removing it when the seed setup changed.